app/controllers/user_controller.php

```diff
@@ -80,5 +80,40 @@ public function logout()
     public function index()
     {
     }
+
+    public function profile()
+    {
+        $user = User::get();
+        $this->set(get_defined_vars());
+    }
```

Comment:
You could just render the view of this method po ..
just make sure that the controller that you use to render the view of this method contains L86-87 :bow: